### ISSUE TITLE
Validation of simple parameters

### ISSUE THREAD

**Issue by tksd123** [pb33f/libopenapi-validator#54]
Hi there, I think I'm facing an issue with validation.

Example like the following:

```yaml
- in: query
  name: count
  description: number of facts to return
  required: false
  schema:
    type: integer
    format: int32
    minimum: 1
```

There is no check on the minimum value in that case with simple schema having a primitive type like integer.

When it is working with:

```yaml
ListSpacesPage:
  name: "page"
  in: "query"
  style: "deepObject"
  description: "list spaces paginated request"
  required: false
  schema:
    type: object
    properties:
      size:
        type: integer
        description: The maximum number of items to return. The service may return fewer than this value.
        minimum: 1
        maximum: 50
        default: 30
        format: int32
        require: false
```

Is this by design, a bug, or I'm missing something?

Cheers and thank you for this amazing lib and the work you're putting in.

**Comment by daveshanley** [comment]
Before I can be helpful, I need to know how you're calling this, what are you validating? a Parameter? a schema? a header? a request? Can you share a code snippet?

**Comment by tksd123** [comment]
hey there, thanks for your quick reply.

We've been trying to validate a query param. I've shared the OAS chunk above where you can see "in: query".
We are simply using the highLevelValidator.ValidateHttpRequest(req)

thanks for your help

**Comment by daveshanley** [comment]
I am not sure, I will have to investigate, it's been a hot minute since I looked at that code.

**Comment by emilien-puget** [comment]
i made https://github.com/pb33f/libopenapi-validator/pull/55 for this issue

**Comment by daveshanley** [comment]
Fixed by @emilien-puget in `v0.0.43`
